cmd/cosign/cli/options/certificate.go

```diff
@@ -37,7 +37,7 @@ var _ Interface = (*RekorOptions)(nil)
 // AddFlags implements Interface
 func (o *CertVerifyOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.Cert, "certificate", "",
-		"path to the public certificate")
+		"path to the public certificate. The certificate will be verified against the Fulcio roots if the --certificate-chain option is not passed.")
```

Comment:
nit: Fulcio root provided in the TUF root?

Reply:
I didn't mention it to TUF here because I have a concern about the description would be too long. If we should mention it, please let me know so I fix it.